spec/components/tabular_card_component_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe TabularCardComponent, type: :component do
+  subject(:component) do
+    described_class.new(title: title, table_content: table_content)
+  end
+
+  let(:rendered) { render_inline(component) }
+  let(:title) { 'My title' }
+  let(:table_content) { { Test: 'Hans' } }
+
+  it 'renders id and title' do
+    expect(rendered.css('h4').text).to eq title
+    expect(rendered.css('.mb-lg-1.col-12.col-xxl-5.font-weight-bold').text).to eq table_content.keys.first.to_s
+    expect(rendered.css('.mb-1.col-12.col-xxl-7').text).to eq table_content.values.first.to_s
```

Comment:
I'd prefer to not use css classes for testing since it's only purpose should be styling. If we decide to have a greater margin, then tests would suddenly fail. I'd suggest to use data attributes (like I did in the cards component, e.g. `.mb-lg-1.col-12.col-xxl-5.font-weight-bold[data-content]` and then use
```suggestion
    expect(rendered.css('[data-content]').text).to eq table_content.keys.first.to_s
    expect(rendered.css('.mb-1.col-12.col-xxl-7').text).to eq table_content.values.first.to_s
```
in specs and/or JavaScript)

Reply:
See also https://github.com/my-zivi/my-zivi/blob/develop/spec/components/overview_card_component_spec.rb#L20 or https://github.com/my-zivi/my-zivi/blob/develop/spec/components/overview_card_component_spec.rb#L33

Reply:
I wasn't very happy either that sounds smart. Why not an id?

Reply:
Because you render your card component multiple times and id's are unique (It's actually not valid XHTML. Also a selector would be ambiguous). data attributes can appear multiple times as you please

Reply:
![image](https://user-images.githubusercontent.com/8350985/87826947-16110180-c87a-11ea-9c86-59b20e842785.png)

Reply:
True true